test/acceptance/tests.py

```diff
@@ -169,6 +172,9 @@ def test_student_training(self):
 
             self.student_training_page.wait_for_page().wait_for_response().assess(options_selected)
 
+            # Check browser scrolled back to top only on first example
+            self.assertEqual(self.self_asmnt_page.is_on_top, example_num == 0)
```

Comment:
@La0 this approach happen to work fine and test the fix, but it would be good to actually verify that correct "Learn to Assess Responses" block is shown. It's a nice to have though, so if it's too onvolving to implement - don't waste too much time on it.